src/WebJobs.Script/CloudBlockBlobHelperService.cs

```diff
@@ -4,19 +4,22 @@
 using System;
 using System.Threading.Tasks;
 using Microsoft.Azure.Storage.Blob;
-using Microsoft.Extensions.Logging;
 
-namespace Microsoft.Azure.WebJobs.Script.WebHost.Management.LinuxSpecialization
+namespace Microsoft.Azure.WebJobs.Script
 {
-    public class RunFromPackageCloudBlockBlobService
+    public class CloudBlockBlobHelperService
     {
-        public virtual async Task<bool> BlobExists(string url, string environmentVariableName, ILogger logger)
+        public virtual async Task<bool> BlobExists(string url)
         {
-            return await BlobExistsAsync(url, environmentVariableName, logger);
+            return await BlobExistsAsync(url);
         }
 
-        private static async Task<bool> BlobExistsAsync(string url, string environmentVariableName, ILogger logger)
+        private static async Task<bool> BlobExistsAsync(string url)
         {
+            if (string.IsNullOrEmpty(url))
+            {
```

Comment:
move this check to public method

Reply:
Sorry, not sure I understand your comment?

Reply:
Oh, you mean the public method above? Sure, let me do that.